.gitignore

```diff
@@ -1,42 +1,17 @@
 !/.idea/codeStyleSettings.xml
 !/.idea/modules.xml
-!/aws/templates/
 .DS_Store
-.grunt
-.Icon
-.vscode
-*.rej
-/_SpecRunner.html
 /.idea/**/*.xml
 /app/ext
-/app/min
 /app/i18n/**/*
 !/app/i18n/**/*.json
-/aws/s3
-/aws/static
-/aws/version
-/aws/routes
-/aws/config.*
-/aws/index.*
-/aws/Server.*
-/aws/ServerConfig.*
-/aws/util
-/bower_components
-/debug.log
-/deploy
-/dist
 /doc
 /docs/auth-coverage
 /docs/coverage
 /keys
-/nbproject
-/node_modules
+node_modules
 /package-lock.json
 /tags
-/temp
```

Comment:
`temp` should be ignored and is application specific (we put translations in there)